src/main/java/com/corundumstudio/socketio/ClientOperations.java

```diff
@@ -38,6 +38,12 @@ public interface ClientOperations {
      */
     void disconnect();
 
+    /**
+     * Disconnect client
```

Comment:
What's different with `disconnect()` ? it's possible to add this change to `disconnect` ?